src/qutip_qip/device/cavityqed.py

```diff
@@ -255,3 +128,152 @@ def load_circuit(self, qc, schedule_mode="ASAP", compiler=None):
         )
         self.global_phase = compiler.global_phase
         return tlist, coeff
+
+
+class CavityQEDModel(Model):
+    """
+    The physical model for a dispersive cavity-QED system.
+
+    Parameters
+    ----------
+    num_qubits : int
+        The number of component systems.
```

Comment:
```suggestion
        The number of component systems.
```
Sounds a bit strange. "The number of qubits"? or "The number of qubits composing the system"?